templates/common/_base/units/ovs-vswitchd.service.yaml

```diff
@@ -0,0 +1,8 @@
+name: ovs-vswitchd.service
```

Comment:
is this a standalone service or is it `openvswitch.service`?

Reply:
if it's openvswitch, this won't work as you're not dropping an override for the correct service, and you'd need something like https://github.com/openshift/machine-config-operator/pull/1927/files#r455735851 anyway

Reply:
ovs-vswitchd.service is a separate service. it's not a drop-in for openvswitch.service.